SPDX-FileCopyrightText: 2023 fosslinux <fosslinux@aussies.space>

SPDX-License-Identifier: PSF-2.0

Install libraries with 755 instead of 555 so we can strip them. (This
is what is in modern versions of python).

--- Makefile.pre.in	2023-03-15 21:49:08.274186777 +1100
+++ Makefile.pre.in	2023-03-15 21:50:02.466143662 +1100
@@ -54,8 +54,7 @@
 INSTALL_DATA=	@INSTALL_DATA@
 # Shared libraries must be installed with executable mode on some systems;
 # rather than figuring out exactly which, we always give them executable mode.
-# Also, making them read-only seems to be a good idea...
-INSTALL_SHARED= ${INSTALL} -m 555
+INSTALL_SHARED= ${INSTALL} -m 755
 
 MAKESETUP=      $(srcdir)/Modules/makesetup
 
